packages/asset-kit/src/components/ThumbnailItem.tsx

```diff
@@ -0,0 +1,67 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useState } from 'react';
+import { useFocusRing } from '@react-aria/focus';
+import { useKeyboard } from '@react-aria/interactions';
+import { mergeProps } from '@react-aria/utils';
+import { ThumbnailItemProps } from '../types';
+import { ThumbnailToolbar } from './';
+import { LoadingCircle } from '@frontify/fondue';
+export const ThumbnailItem = ({
+    asset,
+    isEditing,
+    appBridge,
+    onRemoveAsset,
+    onReplaceAsset,
+    thumbnailStyle,
+}: ThumbnailItemProps) => {
+    const { isFocused, focusProps } = useFocusRing();
+    const [isUploading, setIsUploading] = useState(false);
+
+    const { keyboardProps } = useKeyboard({
+        onKeyUp: (event) => {
+            if ((event.key === 'Backspace' || event.key === 'Delete') && isEditing && isFocused) {
+                onRemoveAsset(asset.id);
+            }
+        },
+    });
+
+    return (
+        <div
+            {...mergeProps(focusProps, keyboardProps)}
+            tabIndex={0}
+            data-test-id="block-thumbnail"
+            className="tw-aspect-square tw-group tw-relative tw-outline-1 tw-outline-offset-1 tw-outline-box-selected-inverse focus:tw-outline hover:tw-outline"
```

Comment:
Maybe check if block is in editing mode and add `hover:tw-outline` only then. Currently the outline on hover is visible even in view mode, which is a bit confusing.

Reply:
you are completely right, i adjusted it.